contact.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <nav id="nav-bar">
+        <a href="index1.html">Home</a>
+        <a href="#">About</a>
+        <a href="contact.html">contact us</a>
+    </nav>
```

Comment:
Kindly use semantic tag (ul) or (ol) on your nav links it makes your code appear more proffessional